testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -427,9 +441,9 @@ impl BitcoinRegtestController {
             return None
         }
         let value = total_unspent - total_spent - tx_fee;
-        if value > DUST_UTXO_LIMIT {
```

Comment:
If the change ouput is not there, this key registration is invalid. The `else` code path should never be executed since this check is already done when we're fetching the utxos, but should we also abort, after logging a warning?

Reply:
The "identifier" output gets added explicitly during the construction of the transaction (it gets pushed to `tx.output` at around line 326), so this check can still be safely performed. The warning we'd see in the log is that `total_unspent` wouldn't be greater than `total_spent + tx_fee`.